src/tools/genpolicy/rules.rego

```diff
@@ -912,10 +912,18 @@ allow_storage_options(p_storage, i_storage, layer_ids, root_hashes) {
 
     p_storage.driver == "smb"
     count(p_storage.options) == 0
```

Comment:
Is this not redundant assuming we trust genpolicy?

Reply:
I agree we don't need to check the count of policy arrays.